quill-core/src/main/scala/io/getquill/quotation/Parsing.scala

```diff
@@ -452,9 +456,14 @@ trait Parsing {
   }
 
   val valueParser: Parser[Ast] = Parser[Ast] {
-    case q"null"                         => NullValue
+    case q"null" => NullValue
+    case q"scala.None" => NullValue
```

Comment:
Let's create a `NoneOption` AST for this

Reply:
Like I said before, do you also want an OptionSome(ast:Ast) as well?